app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class IngredientsController < ApplicationController
+    def index
+        @chef = Chef.find(params[:chef_id])
+        @ingredients = @chef.ingredients.distinct
```

Comment:
This is a minor MVC infraction (and the rubric allows for one) but doing `distinct` in the controller is an AR method beyond simple CRUD actions (.create, .find, .update, .destroy, .save,etc) so it can go in a model method. IF you have this model method in your chef model:
```
def ingredients_list
     ingredients.distinct
end
```
then you can call `@chef.ingredients_list` either in the controller or from the view as well!